charts/fybrik/templates/manager-deployment.yaml

```diff
@@ -42,11 +42,11 @@ spec:
           command:
           - cp 
           - -rf
-          - "/data/tools"
-          - "/tmp/solver/"
+          - {{ include "fybrik.getDataDir" . }}/tools
```

Comment:
Leave "/data/tools" - this is a directory inside the solver container

Reply:
I fixed that. Thanks